articles/asdf-vim-plugin.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "asdf プラグインの作成 : Vimの場合"
+emoji: "🛠"
+type: "tech" # tech: 技術記事 / idea: アイデア
+topics: [vim, asdf]
+published: true
+---
+
+この記事は [asdf プラグインの作成 : Vimの場合 - Qiita](https://qiita.com/tsuyoshi_cho/items/495f081117253f0b23bb) と同じものです。
+
+----
+
+# asdf-vm プラグインの作成
+
+[asdf-vm](https://github.com/asdf-vm/asdf) はバージョニング管理ツール(でいいのかな?)で、[プラグイン](https://github.com/asdf-vm/asdf-plugins) による複数種のツールの管理、また同機構での拡張ができるのが特徴です。
```

Comment:
**[textlint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Disallow to use "?". (ja-technical-writing/no-exclamation-question-mark)